src/Tools/MicroBuild/copy-insertion-items.ps1

```diff
@@ -0,0 +1,50 @@
+Param(
+    [string]$binariesPath = $null,
+    [switch]$test
+)
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+try
+{
+    $items = @(
+        "ExpressionEvaluatorPackage.vsix",
+        "Microsoft.VisualStudio.LanguageServices.Telemetry.vsix",
+        "Microsoft.VisualStudio.VsInteractiveWindow.vsix",
+        "Roslyn.VisualStudio.InteractiveComponents.vsix",
+        "Roslyn.VisualStudio.Setup.Interactive.vsix",
+        "Roslyn.VisualStudio.Setup.Next.vsix",
+        "Roslyn.VisualStudio.Setup.vsix",
+        "Microsoft.CodeAnalysis.ExpressionEvaluator.json",
+        "Microsoft.CodeAnalysis.VisualStudio.Setup.Interactive.json",
+        "Microsoft.CodeAnalysis.VisualStudio.Setup.json",
+        "Microsoft.CodeAnalysis.VisualStudio.Setup.Next.json",
+        "Microsoft.CodeAnalysis.VisualStudio.Telemetry.json",
+        "Microsoft.CodeAnalysis.VisualStudioInteractiveComponents.json",
+        "Microsoft.CodeAnalysis.VisualStudioInteractiveWindow.json",
+        "Microsoft.CodeAnalysis.LanguageServices.vsman",
+        "Microsoft.CodeAnalysis.Compilers.json",
+        "Microsoft.CodeAnalysis.Compilers.vsix",
+        "Microsoft.CodeAnalysis.Compilers.vsman")
+    $destPath = join-path $binariesPath "Insertion"
+    foreach ($item in $items) 
+    {
+        $sourcePath = join-path $binariesPath $item
+
```

Comment:
Please add a log function that prints what is getting copied, it is useful for tracking what should happen in the test build (and what actually happened in the signed build).